fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -1201,9 +1201,6 @@ public void setEnableLeftZigZag(boolean enableLeftZigZag) {
     @VariableMgr.VarAttr(name = DISABLE_NEREIDS_RULES, needForward = true)
     private String disableNereidsRules = "";
 
-    @VariableMgr.VarAttr(name = ENABLE_NEREIDS_RULES, needForward = true)
-    public String enableNereidsRules = "";
-
```

Comment:
not remove it directly, use varType REMOVED to avoid set failed

Reply:
have get it back and use varType to ignore it